server/prisma/schema.prisma

```diff
@@ -8,28 +8,109 @@ datasource db {
   url      = "file:./dev.db"
 }
 
-// This comes with the boilerplate.
 model User {
-  id                  String          @id @default(cuid())
-  createdAt                        DateTime        @default(now())
-  updatedAt                        DateTime        @updatedAt
-  email           String  @unique
-  password      String
-  role          String
-  lastLogin     DateTime?  
-  // LL: https://www.smashingmagazine.com/2017/11/safe-password-resets-with-json-web-tokens/
-  resetPasswordToken String
-  dateResetPasswordRequest DateTime?
-  validateEmailToken String
-  isEmailValidated Boolean
-  name  String
+  id                        Int                         @id @default(autoincrement())
+  firstName                 String
+  lastName                  String
+  email                     String                      @unique
+  feed                      Feed[]
+  board                     Board[]
+  post                      Post[]
```

Comment:
post does not belong to a user -- it belongs to a Source. A source is like Sberbank blog, with many posts.